package.json

```diff
@@ -8,7 +8,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.2",
     "@fortawesome/free-solid-svg-icons": "^5.2.0",
     "@fortawesome/react-fontawesome": "^0.1.0",
-    "@gen3/ui-component": "0.0.2",
+    "@gen3/ui-component": "0.0.4",
```

Comment:
I'm having issues with this version - says it doesn't exist

Reply:
No It doesn't yet. Will pass Travis after merging PR in `ui-component` repo and publishing v0.0.4.